docs/client/README.md

```diff
@@ -66,18 +67,24 @@ Class | Method | HTTP request | Description
 *ConnectionsApi* | [**GetConnections**](docs/ConnectionsApi.md#getconnections) | **Get** /networks/{networkId}/connections | List network connections
 *ConnectionsApi* | [**UpdateConnection**](docs/ConnectionsApi.md#updateconnection) | **Put** /connections/{connectionId} | Update connection
+*FacilitiesApi* | [**FindFacilities**](docs/FacilitiesApi.md#findfacilities) | **Get** /facilities | List facilities
+*FacilitiesApi* | [**GetFacility**](docs/FacilitiesApi.md#getfacility) | **Get** /facilities/{facilityId} | Get facility details
 *LocationsApi* | [**FindLocations**](docs/LocationsApi.md#findlocations) | **Get** /locations | List locations
 *LocationsApi* | [**GetLocation**](docs/LocationsApi.md#getlocation) | **Get** /locations/{locationId} | Get location details
-*NetworksApi* | [**AddConnection**](docs/NetworksApi.md#addconnection) | **Post** /networks/{networkId}/connections | Add new connection
 *NetworksApi* | [**AddNetwork**](docs/NetworksApi.md#addnetwork) | **Post** /accounts/{accountId}/networks | Add new network
 *NetworksApi* | [**DeleteNetwork**](docs/NetworksApi.md#deletenetwork) | **Delete** /networks/{networkId} | Delete network
 *NetworksApi* | [**FindNetworks**](docs/NetworksApi.md#findnetworks) | **Get** /accounts/{accountId}/networks | List networks
-*NetworksApi* | [**GetConnections**](docs/NetworksApi.md#getconnections) | **Get** /networks/{networkId}/connections | List network connections
 *NetworksApi* | [**GetNetwork**](docs/NetworksApi.md#getnetwork) | **Get** /networks/{networkId} | Get network details
 *NetworksApi* | [**UpdateNetwork**](docs/NetworksApi.md#updatenetwork) | **Put** /networks/{networkId} | Update network
 *OptionsApi* | [**GetOptions**](docs/OptionsApi.md#getoptions) | **Get** /options | Get available options
+*PortsApi* | [**AddPort**](docs/PortsApi.md#addport) | **Post** /accounts/{accountId}/ports | Add new port
+*PortsApi* | [**DeleteNetwork1**](docs/PortsApi.md#deletenetwork1) | **Delete** /ports/{portId} | Delete port
+*PortsApi* | [**FindPorts**](docs/PortsApi.md#findports) | **Get** /accounts/{accountId}/ports | List ports
+*PortsApi* | [**GetPort**](docs/PortsApi.md#getport) | **Get** /ports/{portId} | Get port details
+*PortsApi* | [**UpdateNetwork1**](docs/PortsApi.md#updatenetwork1) | **Put** /ports/{portId} | Update port
 *SupportedConnectionsApi* | [**GetSupportedConnection**](docs/SupportedConnectionsApi.md#getsupportedconnection) | **Get** /supportedConnections/{supportedConnectionId} | Get supported connection details
-*SupportedConnectionsApi* | [**GetSupportedConnections**](docs/SupportedConnectionsApi.md#getsupportedconnections) | **Get** /accounts/{accountId}/supportedConnections | List supported connections
+*SupportedConnectionsApi* | [**GetSupportedConnections1**](docs/SupportedConnectionsApi.md#getsupportedconnections1) | **Get** /accounts/{accountId}/supportedConnections | List supported connections
```

Comment:
Referring to your PR to change Kato names, you missed this one.

Reply:
Yeah, not sure what is going on with this one. This is the only function for `GetSupportedConnections` and SwaggerGen in Kato is creating the number.
I can hand modify these for now though:
```
                "description": "",
                "operationId": "getSupportedConnections_1",
                "parameters": [
```

Reply:
Actually, it looks like there is an `AccountSupportedconnections` but is not included in the SwaggerAPI.

Reply:
Should be fix!